src/org/intellij/erlang/quickfixes/ErlangExportFunctionFix.java

```diff
@@ -35,6 +35,7 @@
 import com.intellij.ui.components.JBList;
 import com.intellij.ui.popup.HintUpdateSupply;
 import com.intellij.util.containers.ContainerUtil;
+import kotlin.reflect.jvm.internal.impl.utils.SmartList;
```

Comment:
That's wrong SmartList, be careful. We should use com.intellij.util.SmartList.

Reply:
sorry fixed